src/lib/components/CopyCode/styles.scss

```diff
@@ -1,44 +1,24 @@
 @import "../../../styles/settings/colors";
-@import '../../../styles/settings/global';
 
 .web-copy-code__button {
```

Comment:
does this end up mixing in `base-button` anyway, so we keep the `-webkit-tap-highlight-color: ...` property?

Reply:
Oops, need to import mixins, not these. Good catch.